server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/Bounce.java

```diff
@@ -103,77 +106,55 @@
  */
 public class Bounce extends AbstractNotify {
 
-    /**
-     * Return a string describing this mailet.
-     *
-     * @return a string describing this mailet
-     */
+    private static final String[] CONFIGURABLE_PARAMETERS = new String[] {
+            "debug", "passThrough", "fakeDomainCheck", "inline", "attachment", "message", "notice", "sender", "sendingAddress", "prefix", "attachError" };
+    private static final List<MailAddress> RECIPIENTS = ImmutableList.of(SpecialAddress.REVERSE_PATH);
+    private static final InternetAddress[] TO = new InternetAddress[] { SpecialAddress.REVERSE_PATH.toInternetAddress() };
+
     @Override
     public String getMailetInfo() {
         return "Bounce Mailet";
     }
 
-    /**
-     * Gets the expected init parameters.
-     */
     @Override
     protected String[] getAllowedInitParameters() {
-        return new String[]{
-                // "static",
-                "debug", "passThrough", "fakeDomainCheck", "inline", "attachment", "message", "notice", "sender", "sendingAddress", "prefix", "attachError",};
+        return CONFIGURABLE_PARAMETERS;
     }
 
-    /**
-     * @return <code>SpecialAddress.REVERSE_PATH</code>
-     */
     @Override
     protected Collection<MailAddress> getRecipients() {
-        Collection<MailAddress> newRecipients = new HashSet<MailAddress>();
-        newRecipients.add(SpecialAddress.REVERSE_PATH);
-        return newRecipients;
+        return RECIPIENTS;
     }
 
-    /**
-     * @return <code>SpecialAddress.REVERSE_PATH</code>
-     */
     @Override
     protected InternetAddress[] getTo() {
-        InternetAddress[] apparentlyTo = new InternetAddress[1];
-        apparentlyTo[0] = SpecialAddress.REVERSE_PATH.toInternetAddress();
-        return apparentlyTo;
+        return TO;
     }
 
-    /**
-     * @return <code>SpecialAddress.NULL</code> (the meaning of bounce)
-     */
     @Override
     protected MailAddress getReversePath(Mail originalMail) {
         return SpecialAddress.NULL;
     }
 
-    /**
-     * Service does the hard work,and redirects the originalMail in the form
-     * specified. Checks that the original return path is not empty, and then
-     * calls super.service(originalMail), otherwise just returns.
-     *
-     * @param originalMail the mail to process and redirect
-     * @throws MessagingException if a problem arises formulating the redirected mail
-     */
     @Override
     public void service(Mail originalMail) throws MessagingException {
         if (originalMail.getSender() == null) {
-            if (isDebug)
-                log("Processing a bounce request for a message with an empty reverse-path.  No bounce will be sent.");
-            if (!getPassThrough(originalMail)) {
-                originalMail.setState(Mail.GHOST);
+            passThrough(originalMail);
+        } else {
+            if (isDebug) {
```

Comment:
Do we really need the is debug parameter ? Can't this be handled by the Logger directly ?